### DISCUSSION TITLE
Use cronjobs in Termux

### DISCUSSION THREAD

**Discussion by Jumbo125** [termux/termux-app#3297] [Q&A] [answered]
Sorry to bother you again.
Is it possible to use cronjobs on the apache server?

i found the cronie pkg. is this the perfect solution?
pkg install cronie termux-services
sv-enable crond
crontab -e 

i need to run my php script every night at 23:59

it would be nice, if there as a directory, where i can save the crontab or any gui....
thanks for your help

**Comment by sylirre** [comment, marked as answer]
Use `crontab -e` command to create or edit a crontab.

Note that wake lock should be enabled and Termux app should be kept open. Otherwise `crond` eventually will be terminated by Android.
